splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/joins/BroadcastJoinIT.java

```diff
@@ -631,9 +631,9 @@ public void testUncomparableCharColumnsBroadCastJoin() throws Exception {
                 " on s1.b1 = s2.a2" , useSpark
```

Comment:
Please change the name of this test since it's no longer 'uncomparable'

Reply:
Changed Uncomparable to Comparable.